### PR TITLE
13 feature data access classes

### PR DESCRIPTION
**Pull request by jempio**

### CODE REVIEW

**Review by lucasprates74 — APPROVED** [review]
what is done so far looks good. look into not having to rewrite the entire csv if possible

**Review by Junyu11111 — CHANGES_REQUESTED** [review]
Add a method to get the last ID in FlashcardDataAccess and FlashcardSetDataAccess.

**Review by ThomasShimUoft — CHANGES_REQUESTED** [review]
Implement FlashcardSetDataAccess Methods.

**Review by uoftguy — COMMENTED** [review]
For user, another attribute would be isAdmin, dictating whether or not an user has admin level access or not, it would be between the user password and flashcardSetIds

File: src/main/java/dataAccess/CommonUserDataAccess.java [hunk 1]
```diff
@@ -55,7 +55,7 @@ private void save() {
             writer.newLine();
 
             for (CommonUserDsRequestModel user : accounts.values()) {
-                String line = String.format(user.getUsername(), user.getPassword(), user.getFlashcardSetIds());
+                String line = String.format("%s", "%s", "%s", "%s", user.getUsername(), user.getPassword(), user.getFlashcardSetIds());
```
Comment (uoftguy) [inline src/main/java/dataAccess/CommonUserDataAccess.java:58]: Need to add user.getIsAdmin

**Review by ThomasShimUoft — APPROVED** [review]

**Pull request merged**